tsdb/querier_test.go

```diff
@@ -1168,6 +1168,7 @@ func (m mockIndex) LabelValues(name string, matchers ...*labels.Matcher) ([]stri
 	for _, series := range m.series {
 		for _, matcher := range matchers {
 			if matcher.Matches(series.l.Get(matcher.Name)) {
+				// TODO(colega): shouldn't we check all the matchers before adding this to the values?
```

Comment:
We have N matchers here and we saw that series matches one of them, so we're putting it into values, that's wrong, right?

This is the mock index so 🤷 , but if you confirm, I'll fix it.